models/dev/titanic/titanic.R

```diff
@@ -0,0 +1,125 @@
+# A trivial titanic regression model
+
+# A syberia model file is a nested list structure. Top-level lists are called
+# stages. You can create your own stages by writing `lib/stages/my_stage.R`.
+# A stage should return a [stagerunner](github.com/robertzk/stagerunner) object.
+
+list(
+  # Titanic dataset is fairly popular. Here we're downloading it from a public github repo
+  import = list(
+    # File, R and s3 adapters ship by default. If you want to make a different adapter
+    # you can define one by writing `lib/adapters/my_adapter.R`. Here we have made
+    # a custom URL adapter
+    url = 'https://raw.githubusercontent.com/haven-jeon/introduction_to_most_usable_pkgs_in_project/master/bicdata/data/titanic.csv'
+  ),
+
+  # Data stage is a perfect place to transform your dataset prior to modeling
+  # The default data stage defines a DSL for creating and training
+  # [mungebits](github.com/robertzk/mungebits)
+  # Yes, you need to train your data preparation!
+  # Traditionally data scientists have been preparing models and shipping them to
+  # engineers that would reimplement them in Java or another traditional server language.
+  # This is a very slow and extremely error-prone process.
+  #
+  # Also, there is one more important consideration: data preparation should
+  # operate differently in train versus predict!
+  # For example, let's say that we want to impute a missing variable using column mean.
+  # In training, you'd want to use the mean calculated from the import stage dataframe.
+  # However, in production you do not have access to the input dataframe anymore!
+  # So you need to store the imputed mean somewhere and use that number in production.
+  # Data stage takes care of this duality, allowing you to use a plethora of mungebits
+  # from [syberiaMungebits](github.com/robertzk/syberiaMungebits). Or you can write your own
+  # and put them in `lib/mungebits/my_mungebit.R`
+  data = list(
+    # The left-hand side defines the informal name of a mungebit that you will see
+    # when you run this model.
+    # The right-hand side is the mungebit invocation.
+    "Factor to character"   = list(column_transformation(as.character), is.factor)
+
+    # Both `column_transformation` and `multi_column_transformation` come from [syberiaMungebits] package
+    # This particular mungebit creates a new variable - *name_length*
+    ,"Name length variable" = list(multi_column_transformation(function(name) nchar(name)), "name", "name_length")
```

Comment:
Instead of using `multi_column_transformation`, maybe use `new_variable` mungebit:

```R
list(new_variable, function(name) nchar(name), "name_length")